src/utils/urls.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * Gets the final URL needed to load an IFC model
```

Comment:
So am I understanding correctly that large files are hosted on a separate server, and this code detects the size difference and redirects the query to that server?

Reply:
This code grabs the first 64 bytes of the file.

If that initial content contains a LFS signature, then it'll download from the Github LFS servers.